db/seeds/posts/global_faq.html

```diff
@@ -1,18 +1,13 @@
-<h2 id="what-are-trust-levels-and-how-are-they-calculated-">What are trust levels and how are they calculated?</h2>
-<!-- I'm omitting any discussion of reputation for now since consensus seems to be that it is beyond MVP. If that is implemented we can add information here. -->
-<p>Trust levels are the system Codidact uses to determine user privileges. Everyone starts at Trust Level 0 and gradually gains the ability to contribute at a higher level. Trust levels are earned by positively contributing to the site. For more information, see <a href="https://github.com/codidact/docs/wiki/User-Privileges">https://github.com/codidact/docs/wiki/User-Privileges</a>. </p>
-<h2 id="what-does-it-mean-that-codidact-is-community-controlled-what-makes-it-different-from-any-other-q-a-site-">What does it mean that Codidact is community-controlled? What makes it different from any other Q&amp;A site?</h2>
-<p>Most Q&amp;A sites are run by companies whose primary goal is to make a profit for their shareholders. This leads to a situation where bringing in revenue can be more important than the health of the community using the site. On Codidact, the users control the communities they use, so there&#39;s no conflict of interest between quality and the bottom line.</p>
+<p>You will need to register an account. Make sure you are logged into the appropriate topic-related site for your question and click the &quot;Ask Question&quot; button at the top of the page. A template should appear with helpful guidelines on how to ask a quality question. Check out the local FAQ page for important information about each community.</p>
+
+<!-- If you change the default, update this section. -->
 <h2 id="what-license-do-my-posts-fall-under-">What license do my posts fall under?</h2>
 <p>Most posts fall under <a href="https://creativecommons.org/licenses/by-sa/4.0/">CC BY-SA 4.0</a>. Some communities or contributors may use a license that more closely aligns with the content posted or their ideological position. Check a community&#39;s local FAQ page and Terms of Service for site-specific details. </p>
-<h2 id="how-can-i-contribute-">How can I contribute?</h2>
-<!-- Eventually if/when we donations we should add a link here -->
-<p>The best way to contribute is by joining a community and submitting quality questions and answers! If you want to join our technical team, check out our GitHub site <a href="https://www.github.com/codidact">here</a> to get started. </p>
+
+
+<h2 id="what-is-codidact-">What is Codidact?</h2>
+<p>This community is built using <a href="https://github.com/codidact/qpixel">software</a> provided by <a href="https://codidact.org">The Codidact Foundation</a>. The software is free, open-source, and customizable.</p>
```

Comment:
Perhaps you can include the name of QPixel here and indicate that it is open source? That makes the link to the github also more sensible.

```suggestion
<p>This community is built using the <a href="https://github.com/codidact/qpixel">open-source QPixel software</a> provided by <a href="https://codidact.org">The Codidact Foundation</a>. The software is free, open-source, and customizable.</p>
```